src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -2112,33 +2038,23 @@ private void addEmailNamePairsToTable(Map<String, String> emailNameTable,
     private List<FeedbackSessionDetailsBundle> getFeedbackSessionDetailsForCourse(
             String courseId, String instructorEmail)
             throws EntityDoesNotExistException {
-
-        List<FeedbackSessionDetailsBundle> fsDetailsWithoutPrivate = new ArrayList<>();
+        List<FeedbackSessionDetailsBundle> fsDetails = new ArrayList<>();
         List<FeedbackSessionAttributes> fsInCourse =
                 fsDb.getFeedbackSessionsForCourse(courseId);
 
         for (FeedbackSessionAttributes fsa : fsInCourse) {
-            if (!fsa.isPrivateSession() || fsa.isCreator(instructorEmail)) {
-                fsDetailsWithoutPrivate.add(getFeedbackSessionDetails(fsa));
+            if (fsa.isCreator(instructorEmail)) {
+                fsDetails.add(getFeedbackSessionDetails(fsa));
```

Comment:
I believe the condition is wrong - the if is no longer necessary. The `isCreator` check was only applicable for private sessions.